frontend/src/pages/Alerts/AlertConfigurationCard/AlertConfigurationCard.tsx

```diff
@@ -930,8 +936,13 @@ export const AlertConfigurationCard = ({
 										<TextTransition
 											inline
 											text={
-												frequency === '1' ||
-												frequency === '60'
+												[
+													'1',
+													'60',
+													'3600',
+													'86400',
+													'604800',
+												].includes(frequency)
```

Comment:
is there any way to reference the constants defined in `AlertConfigurationConstants.ts`?